#! /bin/sh /usr/share/dpatch/dpatch-run
## 28_fix_v4l.dpatch by  Michael Haas <laga@laga.ath.cx>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch by David Madsen. See http://svn.mythtv.org/trac/ticket/5608

@DPATCH@
diff -urNad mythtv-fixes~/libs/libmythtv/NuppelVideoRecorder.cpp mythtv-fixes/libs/libmythtv/NuppelVideoRecorder.cpp
--- mythtv-fixes~/libs/libmythtv/NuppelVideoRecorder.cpp	2008-09-20 16:53:49.000000000 +0200
+++ mythtv-fixes/libs/libmythtv/NuppelVideoRecorder.cpp	2008-09-21 19:33:29.000000000 +0200
@@ -981,7 +981,7 @@
             correct_bttv = true;
 
         QString driver = (char *)vcap.driver;
-        if (driver == "cx8800" || driver == "go7007" || driver == "em28xx")
+        //if (driver == "cx8800" || driver == "go7007" || driver == "em28xx")
         {
             channelfd = open(videodevice.ascii(), O_RDWR);
             if (channelfd < 0)
